.gitignore

```diff
@@ -1,5 +1,4 @@
 .DS_Store
 npm-debug.log
 node_modules
-lib
```

Comment:
I think lib files should not be managed in GitHub.
They should be used for the npm package.